### PR TITLE
Initial implementation of Loggly service

### PR DESCRIPTION
**Pull request by darinkrauss**
- Implement Service and ServiceUI protocols
- Implement Logging protocol
- Connect with Loggly third-party service

https://tidepool.atlassian.net/browse/LOOP-22

### CODE REVIEW

**Review by nhamming — DISMISSED** [review]
Similar comments as amplitude. If you agree with them, then consider them here as well. 

Otherwise LGTM.

**Review by nhamming — DISMISSED** [review]
+1

**Review by nhamming — DISMISSED** [review]
+1

File: LogglyServiceKitUI/LogglyServiceTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+extension AuthenticationTableViewCell: NibLoadable {}
+
+extension TextButtonTableViewCell: IdentifiableClass {}
+
+fileprivate extension UIAlertController {
+    
+    convenience init(serviceDeletionHandler handler: @escaping () -> Void) {
+        self.init(
+            title: nil,
+            message: NSLocalizedString("Are you sure you want to delete this service?", comment: "Confirmation message for deleting a service"),
```
Comment (ps2) [inline LogglyServiceKitUI/LogglyServiceTableViewController.swift]: NSLocalizedString -> LocalizedString

**Review by ps2 — CHANGES_REQUESTED** [review]

**Review by ps2 — APPROVED** [review]
🎉 LGTM!

**Pull request merged**